src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -137,7 +137,7 @@ private void OnViewModelMessageLogged(ILogMessage msg)
 
         private void AddToSidebar(bool displayDefaultContent)
         {
-            // verify the browser window has been initialised
+            // verify the browser window has been initialized
```

Comment:
That's the British spelling. I can see why they spelt it that way :)

Reply:
@aparajit-pratap Oops.. I was not aware of that...